Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ARG image=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:2023-02-22-1677092456.2
-ARG golang_image=public.ecr.aws/eks-distro-build-tooling/golang:1.19-gcc
+ARG golang_image=public.ecr.aws/docker/library/golang:1.21.4
```

Comment:
I think this is fine for community releases, so long as AWS releases override this to use AWS-built Go